Configurations/unix-Makefile.tmpl

```diff
@@ -654,9 +654,11 @@ errors:
 	( cd $(SRCDIR); $(PERL) util/ck_errf.pl -strict */*.c */*/*.c )
 	( cd $(SRCDIR); $(PERL) util/mkerr.pl -internal )
 	( cd $(SRCDIR)/engines; \
-	  for e in *.ec; do \
-	      $(PERL) ../util/mkerr.pl -conf $$e -rebuild *.c; \
-	  done )
+          $(PERL) ../util/mkerr.pl -conf e_capi.ec e_capi.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_dasync.ec e_dasync.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_ossltest.ec e_ossltest.c ; \
+          cd afalg; \
+          $(PERL) ../../util/mkerr.pl -conf e_afalg.ec e_afalg.c )
```

Comment:
It will fail for any engine that has more than one non-error-code C file.  It's not the case for our current source, but you never know when it'll turn up.  I think it *is* better to scan too many C files than too few, so I disagree with the basename idea.